doc/source/theory/fluid_dynamics/rheology.rst

```diff
@@ -0,0 +1,51 @@
+Rheology and non-Newtonian fluids
+----------------------------------------
+
+For numerous flows that we encounter in chemical engineering, Newton's equation for viscosity is insufficient to describe the rheological behavior of the fluids. For these problems, the viscosity is not constant anymore. Lethe supports the simulation of non-Newtonian fluids by using generalized newtonian models. In these models, the viscosity depends on the magnitude of the shear-rate tensor :math:`\dot{\gamma}`. In this case, the viscosity cannot be assumed to be constant anymore and the incompressible Navier-Stokes equations take the following form:
+
+.. math::
+    \partial_j u_j &= 0  \\
+    \partial_i+ u_i \partial_i u_j &= -\frac{1}{p} \partial_i p  -  \partial_j \mathbf{\tau}_{ji}
```

Comment:
```suggestion
    \partial_i+ u_i \partial_i u_j &= -\frac{1}{\rho} \partial_i p  -  \partial_j \mathbf{\tau}_{ji}
```

Reply:
Maybe we could standardize and only use l,k or i,j for indices

Reply:
Yeah I would use {l,k} for indices, because {i,j} are used in the DOF loops in the code and I don't want to confuse people.